extensions/amp-carousel/0.1/tsconfig.json

```diff
@@ -3,8 +3,17 @@
     "allowJs": true,
     "checkJs": true,
     "noEmit": true,
-    "paths": {}
+    "downlevelIteration": true,
+    "target": "esnext",
+    "moduleResolution": "node",
+    "paths": {
+      "#compiler": ["../../../src/compiler"],
+      "#compiler/*": ["../../../src/compiler/*"],
+      "#core": ["../../../src/core"],
+      "#core/*": ["../../../src/core/*"],
+      "#third_party/*": ["../../../third_party/*"]
+    }
   },
-  "include": ["*.js", "**/*.d.ts", "../../../src/*.d.ts"],
-  "exclude": []
+  "include": ["*.js", "**/*.d.ts", "../../../src/**/*.d.ts"],
+  "exclude": ["../../../src/purifier/index.d.ts"]
```

Comment:
Could you still extend the base here, but override the `noImplicitAny`?

Reply:
I'll test it out. The thing I'm more concerned about is `#paths`.

Reply:
Cannot do this yet, as not all of `#utils` is typed yet and this utilizes utils.

Reply:
If we add `.d.ts` files for the `utils` dependency it would make future typechecking progressively easier